Mathlib/MeasureTheory/Constructions/BorelSpace/Basic.lean

```diff
@@ -320,9 +320,10 @@ theorem IsOpen.measurableSet (h : IsOpen s) : MeasurableSet s :=
   OpensMeasurableSpace.borel_le _ <| GenerateMeasurable.basic _ h
 #align is_open.measurable_set IsOpen.measurableSet
 
-instance (priority := 500) {s : Set α} [HasCountableSeparatingOn α IsOpen s] :
-    HasCountableSeparatingOn α MeasurableSet s :=
-  .mono (fun _ ↦ IsOpen.measurableSet) Subset.rfl
+instance (priority := 1000) {s : Set α} [h : HasCountableSeparatingOn α IsOpen s] :
```

Comment:
I increased it because some proofs in Constructions.Polish were timing out. You can see these in the result of CI on [this commit](https://github.com/leanprover-community/mathlib4/pull/12433/commits/73d6643f1fd2596d564b3b1687420ca81ab7a1cd)

I did not put any thought into the specific value other than making it higher. I do think it makes sense for the priority to be quite high. This plus [instHasCountableSeparatingOnIsOpen](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/CountableSeparatingOn.html#instHasCountableSeparatingOnIsOpen) should almost always be how instances of `CountableSeparated` are generated for Borel spaces.